marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/steps/CustomStepController.java

```diff
@@ -42,7 +42,7 @@
     @Secured("ROLE_writeCustom")
     public ResponseEntity<Void> updateCustomStep(@RequestBody @ApiParam(hidden = true) ObjectNode propertiesToAssign, @PathVariable String stepName) {
         propertiesToAssign.put("name", stepName);
-        newStepService().saveStep(STEP_DEFINITION_TYPE, propertiesToAssign, false, false);
+        newService().updateCustomStep(propertiesToAssign);
```

Comment:
The method can be called customStepService().

Reply:
I don't mind newService here, as my first guess is that it's a CustomStepService since this is a CustomStepController. If multiple services were needed, then I agree with using a more descriptive name.